dali/python/setup.py.in

```diff
@@ -14,7 +14,7 @@
 
 from setuptools import setup, find_packages
 
-setup(name='nvidia-dali',
+setup(name='nvidia-dali@DALI_FLAVOR@',
       description='NVIDIA DALI for CUDA @CUDA_VERSION@',
       url='https://github.com/NVIDIA/dali',
```

Comment:
we probably need to maki those nvidia-dali-weekly nvidia-dali-daily and nvidia-dali mutually exclusive (one replaces the other one)

Reply:
Done

Reply:
Done